pdns/namespaces.hh

```diff
@@ -78,4 +78,18 @@ using boost::trim_right_copy_if;
 using boost::equals;
 using boost::ends_with;
 using boost::iends_with;
+
+// apple compiler somehow has string_view even in c++11!
+#if __cplusplus < 201703L && !defined(__APPLE__)
+#if BOOST_VERSION > 105400
+#include <boost/utility/string_view.hpp>
+using boost::string_view;
+#else
+#include <boost/utility/string_ref.hpp>
```

Comment:
Looks like this requires Boost 1.53, we previously required 1.42+ with protobuf. Given that we already have 
a reference to an existing string in every place where we currently use a `string_view`, perhaps we can skip that for now?